server/src/test/integration/drinksEndpointTest.js

```diff
@@ -0,0 +1,62 @@
+const app = require('../../server');
+const request = require('supertest');
+const expect = require('chai').expect;
+const dbAdapter = require('../../db/adapter');
+
+describe(`'drinks' route - api test`, function() {
```

Comment:
Arrow jawns could be used here

Reply:
The [mocha docs](https://mochajs.org/#arrow-functions) suggest not to use arrow functions with `describe` or `it` functions because it screws up mochas context. I'm not using the context but just in case I would prefer to leave these as is for now